### ISSUE TITLE
Retained messages not received from bridge at first startup 

### ISSUE THREAD

**Issue by lrodorigo** [eclipse/mosquitto#701]
I am using mosquitto v1.4.12 in eclipse-mosquitto Docker image.

Hello, I am performing some tests in the following scenario.
I have two brokers, Broker A (in my LAN, no public IP) and Broker B (public IP, deployed on the cloud) connected through bridge functionality (A connects to B).

Broker A configuration:
```
connection cloud
address <server IP>
topic # both 2
bridge_protocol_version mqttv311
```
I am using default values for the other settings.

If I publish a retained message when both Broker A and Broker B are active I am able to receive the message both from clients connected to Broker A and from clients connected to Broker B.

Instead I noticed a problem in the following case.

I shutdown Broker B, publish a retained message on Broker A. Then, I activate Broker B.
Clients that connect to Broker B do not receive the retained message (until I manually re-publish the retained message on A), i.e. it seems that Broker B does not retrieve retained messages from Broker A at startup.

Conversely, in case I shutdown Broker A, publish a retained message on B and, then, activate Broker A again, retained message is received both from clients connected on A and clients connected on B.

Thanks in advance for your help.

**Comment by karlp** [comment]
I'm seeing the same thing with develop branch.  

Broker "local" has outbound, qos1 topic mapping to "remote"
publish retained to local.
restart "remote" broker -> clients connecting to remote don't see the retained messages.

**Comment by ralight** [comment]
This is fixed in my local fixes branch and will become part of 1.5.3 shortly.
